_data/retail.yml

```diff
@@ -36,8 +36,8 @@ websites:
       sms: Yes
       software: Yes
       exceptions:
-          text: "See http://support.apple.com/kb/HT5593 for a list of supported SMS carriers."
-      doc: http://support.apple.com/kb/ht5570
+          text: "See https://support.apple.com/kb/HT5593 for a list of supported SMS carriers."
+      doc: https://support.apple.com/kb/ht5570
```

Comment:
Redirects to `https://support.apple.com/kb/HT204152`

Reply:
updated